mne/decoding/tests/test_csp.py

```diff
@@ -93,3 +93,43 @@ def test_regularized_csp():
         csp.n_components = n_components
         sources = csp.transform(epochs_data)
         assert_true(sources.shape[1] == n_components)
+
+
+@requires_sklearn
+def test_plot_pattern():
+    """Test plot for Common Spatial Patterns
+    """
+    raw = io.Raw(raw_fname, preload=False)
+    events = read_events(event_name)
+    picks = pick_types(raw.info, meg=True, stim=False, ecg=False,
+                       eog=False, exclude='bads')
+    epochs = Epochs(raw, events, event_id, tmin, tmax, picks=picks,
+                    baseline=(None, 0), preload=True)
+    epochs_data = epochs.get_data()
+
+    n_components = 3
+    csp = CSP(n_components=n_components, reg='lws')
+
+    csp.fit(epochs_data, epochs.events[:, -1])
+    csp.plot_patterns(epochs.info)
+    csp.plot_patterns(epochs.info, components=np.arange(10))
+
+
+@requires_sklearn
+def test_plot_filters():
+    """Test plot for Common Spatial Patterns
+    """
+    raw = io.Raw(raw_fname, preload=False)
+    events = read_events(event_name)
+    picks = pick_types(raw.info, meg=True, stim=False, ecg=False,
+                       eog=False, exclude='bads')
+    epochs = Epochs(raw, events, event_id, tmin, tmax, picks=picks,
+                    baseline=(None, 0), preload=True)
+    epochs_data = epochs.get_data()
+
+    n_components = 3
+    csp = CSP(n_components=n_components, reg='lws')
+
+    csp.fit(epochs_data, epochs.events[:, -1])
+    csp.plot_filters(epochs.info)
+    csp.plot_filters(epochs.info, components=np.arange(10))
```

Comment:
it runs pretty quickly... 
line135 have been removed from the test and reg changed to .9
Maybe it is due to some convergence issues in lws